packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -161,6 +165,18 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
             _networkController.clear();
           },
         ),
+        const Expanded(child: SizedBox()),
```

Comment:
can you change the properties on the row instead of adding this degenerate expanded?